scipy/stats/tests/test_distributions.py

```diff
@@ -5143,6 +5143,16 @@ def test_crystalball_function_moments():
     assert_allclose(expected_5th_moment, calculated_5th_moment, rtol=0.001)
 
 
+def test_crystalball_entropy():
+    cb = stats.crystalball(2, 3)
+    res1 = cb.entropy()
+    res2 = quad(lambda x: entr(cb.pdf(x)), -np.inf, np.inf)[0]
+    N, lo, hi = 10000, -100, 20
+    res3 = entr(cb.pdf(np.linspace(lo, hi, N))).sum()*(hi-lo)/N
+    assert_allclose(res1, res2)
+    assert_allclose(res1, res3, rtol=1e-3)
```

Comment:
Maybe add a comment on why the tolerance is only 1e-3

Reply:
I used `trapezoid` to get better accuracy here.